website/content/en/status/report-2023-01-2023-03/pkgbase.live.adoc

```diff
@@ -0,0 +1,24 @@
+=== PkgBase.live
+
+Links: +
+link:https://alpha.pkgbase.live/[Website (defunkt)] URL: link:https://alpha.pkgbase.live/[https://alpha.pkgbase.live/] +
```

Comment:
I'd suggest "defunct" (or "dead").

Reply:
my spellchecker didn't complain, but I'll change it.